traffic_ops/traffic_ops_golang/routing/wrappers.go

```diff
@@ -285,3 +285,11 @@ func NotImplementedHandler() http.Handler {
 		w.Write([]byte(`{"alerts":[{"level":"error","text":"The requested api version is not implemented by this server. If you are using a newer client with an older server, you will need to use an older client version or upgrade your server."}]}`))
 	})
 }
+
+func DisabledRouteHandler() http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		w.Header().Set(tc.ContentType, tc.ApplicationJson)
+		w.WriteHeader(http.StatusServiceUnavailable)
+		w.Write([]byte(`{"alerts":[{"level":"error","text":"The requested route is currently disabled."}]}`))
```

Comment:
pls end with newline

but also, this handler isn't setting headers like e.g. `X-Server-Name` and CORS headers. I don't really know about CORS being important at endpoints that specifically aren't allowed to work, but it could be nice to still see what server served your request.

Reply:
For this handler, I basically just copied the same `NotImplementedHandler` which does the exact same thing except with a different status code and text. I didn't feel like wrapping this in the `wrapHeaders` func to set all the extra headers, gzip, sha, etc. was necessary right now, especially when the `NotImplementedHandler` isn't used that way either. If it becomes necessary for clients in the future, we should add it then IMO.

If you so desire, I'll add a newline though.

Reply:
I desire greatly

Reply:
done